File: Lesson_5/Class_work/Task_1/CalculatorModel.java
package Lesson_5.Class_work.Task_1;

public class CalculatorModel {
    private double result;

    public void add(double number) {
        result += number;
    }

    public void minus(double number) {
        result -= number;
    }

    public void division(double number) {
        CalculatorView localNumber = new CalculatorView();
        while (number == 0) {
            System.out.println("Cannot divide by zero, please enter another number");
            number = localNumber.getDouble_2();
        }
        result /= number;
    }

    public void mult(double number) {
        result *= number;
    }


    public double getResult() {
        return result;
    }

    public void setResult(double result) {
        this.result = result;
    }
}

File: Lesson_7/Home_task/Task_1/CalculatorModel.py
import CalculatorView
class CalculatorModel:
    def __init__(self):
        self.result = 0

    def add(self, number):
        self.result += number

    def minus(self, number):
        print(number)
        a = self.result - number
        self.result = a

    def division(self, number):
        localNumber = CalculatorView()
        while number == 0:
            print("Cannot divide by zero, please enter another number")
            number = localNumber.getDouble_2()
        self.result /= number

    def mult(self, number):
        self.result *= number

    def getResult(self):
        return self.result

    def setResult(self, result):
        self.result = result
